### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by jessegoodier**
fixed permissions on config files and added sample playbook

### CODE REVIEW

**Comment by jessegoodier** [comment]
updated, thanks for reviewing. my first pull request, so let me know if I messed anything up.

File: tasks/configure-app-protect.yml [hunk 1]
```diff
@@ -17,13 +17,15 @@
...
     dest: "{{ app_protect_security_policy_template.out_file_location }}{{ app_protect_security_policy_template.out_file_name }}"
+    mode: "0644"
     backup: true
   when: app_protect_security_policy_template_enable
 
 - name: "Dynamically Generate NGINX App Protect log policy file"
   template:
     src: "{{ app_protect_log_policy_template.template_file }}"
     dest: "{{ app_protect_log_policy_template.out_file_location }}{{ app_protect_log_policy_template.out_file_name }}"
+    mode: '0644'
```
Comment (aknot242) [inline tasks/configure-app-protect.yml:28]: Mind changing the quotes to doubles on this?

Comment (jessegoodier) [reply tasks/configure-app-protect.yml]: done.

**Comment by jessegoodier** [comment]
should be good.

**Pull request merged**